ethcore/client-traits/src/lib.rs

```diff
@@ -173,6 +177,10 @@ pub trait EngineClient: Sync + Send + ChainInfo {
 pub trait ImportBlock {
 	/// Import a block into the blockchain.
 	fn import_block(&self, block: Unverified) -> EthcoreResult<H256>;
+
+	// todo[dvdplm]: probably a bad place for this method
+	/// Triggered by a message from a block queue when the block is ready for insertion
```

Comment:
perhaps document that it returns the number of imported blocks

additionally, the type that implements it needs access to the block queue but it is implied by the method signature I guess.